packages/sources/s3reader/src/endpoint/csv.ts

```diff
@@ -0,0 +1,66 @@
+import { AdapterEndpoint } from '@chainlink/external-adapter-framework/adapter'
+import { InputParameters } from '@chainlink/external-adapter-framework/validation'
+import { SingleNumberResultResponse } from '@chainlink/external-adapter-framework/util'
+import { config } from '../config'
+import overrides from '../config/overrides.json'
+import { transport } from '../transport/csv'
+
+export const inputParameters = new InputParameters(
+  {
+    bucket: {
+      required: true,
+      type: 'string',
+      description: 'The S3 bucket to query',
+    },
+    key: {
+      aliases: ['path'],
+      required: true,
+      type: 'string',
+      description: 'The path of the file stored in S3',
+    },
+    headerRow: {
+      required: true,
+      type: 'number',
+      description: 'The 1-indexed row of the CSV file that contains the column headers',
+    },
+    matcherField: {
+      required: true,
+      type: 'string',
+      description: 'The column field to compare with the matcherValue',
+    },
+    matcherValue: {
+      required: true,
+      type: 'string',
+      description: 'The value to match with matcherField',
+    },
+    resultField: {
+      required: true,
+      type: 'string',
+      description: 'The row of the CSV file to query',
```

Comment:
I think this is a column rather than row
 
```suggestion
      description: 'The column of the CSV file to return a result for, where the row matches matcherColumn and matcherValue',
```

Reply:
Handled in pushed commit